app/views/automated_tests/_form.html.erb

```diff
@@ -7,7 +7,10 @@
   <script>
     document.addEventListener('DOMContentLoaded', () => {
       window.autotestManagerComponent = makeAutotestManager(document.getElementById('autotest-manager'),
-        { assignment_id: <%= @assignment.id %>});
+        { assignment_id: <%= @assignment.id %>,
+          grouping_id: <%= @assignment.groupings.joins(student_memberships: :user)
```

Comment:
don't do this here. Find or create the test student and its grouping and store it as an instance variable on the controller that renders this view. Or, create an association to the solution grouping on the assignment itself.

Also, change the prop name to something more descriptive. ex: `solution_grouping_id`